scipy/stats/_stats_py.py

```diff
@@ -4942,6 +4968,13 @@ def spearmanr(a, b=None, axis=0, nan_policy='propagate',
             `alternative` above for alternative hypotheses. `pvalue` has the
             same shape as `statistic`.
 
+    Raises
+    ------
+    ValueError
+        If `axis` is not 0, 1 or None, or if dimensions of `a` along the `axis`
```

Comment:
```suggestion
        If `axis` is not 0, 1 or None, or if the number of dimensions of `a`
```
Or "dimensionality" of `a`. `axis` isn't part of this consideration.